ush/eva/gen_eva_obs_yaml.py

```diff
@@ -94,5 +110,7 @@ def gen_eva_obs_yaml(inputyaml, templateyaml, outputdir):
     parser.add_argument('-i', '--inputjediyaml', type=str, help='Input JEDI YAML Configuration', required=True)
     parser.add_argument('-t', '--templateyaml', type=str, help='Template EVA YAML', required=True)
     parser.add_argument('-o', '--outputdir', type=str, help='Output directory for EVA YAMLs', required=True)
+    parser.add_argument('-g', '--group', type=str, help='ioda groups [ObsValue, ombg, ...] ', required=False, default='ObsValue')
```

Comment:
I'm not convinced this will reproduce current behavior even with the default/required=False... Can you check from one of the test YAMLs?

Reply:
fixed